core/src/main/scala/kafka/security/auth/Acl.scala

```diff
@@ -24,6 +24,8 @@ import scala.collection.JavaConverters._
 
 object Acl {
   val WildCardPrincipal: KafkaPrincipal = new KafkaPrincipal(KafkaPrincipal.USER_TYPE, "*")
+  val wildCardUserTypePrincipal: KafkaPrincipal = new KafkaPrincipal(KafkaPrincipal.USER_TYPE, "*")
```

Comment:
Looks like above variable and this is the same. Can you not use the above one?

Also, can you make w in the variable name capital (WildCardUserTypePrincipal)

Reply:
ok